src/scripts/main.js

```diff
@@ -1,3 +1,175 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const tableBody = document.querySelector('tbody');
+const tablerows = document.querySelectorAll('tbody tr');
+const tablerowsArray = Array.from(tablerows);
+
+const offices = [`Tokyo`, `Singapore`,
+  `London`, `New York`, `Edinburgh`, `San Francisco`];
+
+const formFields = {
+  'name': 'text',
+  'position': 'text',
+  'office': 'select',
+  'age': 'number',
+  'salary': 'number',
+};
+
+function capitalise(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+let listOrder = true;
+
+table.addEventListener('click', function(evt) {
+  evt.preventDefault();
+
+  if (evt.target.tagName === 'TH') {
+    const clickCol = evt.target.cellIndex + 1;
+
+    const sorted = tablerowsArray.sort((a, b) => {
+      const itemOne = a.querySelector(`td:nth-child(${clickCol}`).innerHTML;
+      const itemTwo = b.querySelector(`td:nth-child(${clickCol}`).innerHTML;
+
+      if (listOrder === true) {
+        return itemOne.localeCompare(itemTwo, 'en', { numeric: 'true' });
+      }
+
+      if (listOrder === false) {
+        return itemTwo.localeCompare(itemOne, 'en', { numeric: 'true' });
+      }
+    });
+
+    listOrder = !listOrder;
+
+    sorted.forEach(row => {
+      tableBody.appendChild(row);
+    });
+  }
+
+  if (evt.target.tagName === 'TD') {
+    evt.stopPropagation();
+    tablerows.forEach(rowClass => rowClass.classList.remove('active'));
+    evt.target.parentNode.classList = 'active';
+    evt.target.contentEditable = true;
+  }
+});
+
+// form
```

Comment:
You can avoid using such comments because the next constant exactly is `form` :)